logging/src/main/java/org/wso2/testgrid/logging/Slf4jSessionLogger.java

```diff
@@ -0,0 +1,186 @@
+/*
+ * Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.wso2.testgrid.logging;
+
+import org.eclipse.persistence.logging.AbstractSessionLog;
+import org.eclipse.persistence.logging.SessionLog;
+import org.eclipse.persistence.logging.SessionLogEntry;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * A wrapper class to log persistence log messages through SLF4J.
+ *
+ * The following property is used in persistence.xml file for using SLF4J
+ * {@code <property name="eclipselink.logging.logger" value="org.wso2.testgrid.logging.Slf4jSessionLogger" />}
+ */
+public class Slf4jSessionLogger extends AbstractSessionLog {
+   public static final String ECLIPSELINK_NAMESPACE = "org.eclipse.persistence.logging";
+   public static final String DEFAULT_CATEGORY = "default";
+   public static final String DEFAULT_ECLIPSELINK_NAMESPACE = ECLIPSELINK_NAMESPACE + "." + DEFAULT_CATEGORY;
+   private static Map<Integer, LogLevel> mapLevels = new HashMap<Integer, LogLevel>();
+
+    /**
+     * Stores all the java.util.logging.Levels with matching Slf4j level.
+     */
+   static {
+      mapLevels.put(SessionLog.ALL, LogLevel.TRACE);
+      mapLevels.put(SessionLog.FINEST, LogLevel.TRACE);
+      mapLevels.put(SessionLog.FINER, LogLevel.TRACE);
+      mapLevels.put(SessionLog.FINE, LogLevel.DEBUG);
+      mapLevels.put(SessionLog.CONFIG, LogLevel.INFO);
+      mapLevels.put(SessionLog.INFO, LogLevel.INFO);
+      mapLevels.put(SessionLog.WARNING, LogLevel.WARN);
+      mapLevels.put(SessionLog.SEVERE, LogLevel.ERROR);
+   }
+   
+   private Map<String, Logger> categoryLoggers = new HashMap<String, Logger>();
+
+   /**
+    * Creates a new instance of Slf4jSessionLogger
+    */
+   public Slf4jSessionLogger() {
+      super();
+       for (String category : SessionLog.loggerCatagories) {
+           addLogger(category, ECLIPSELINK_NAMESPACE + "." + category);
+       }
+       addLogger(DEFAULT_CATEGORY, DEFAULT_ECLIPSELINK_NAMESPACE);
+   }
+
+   /**
+    * {@inheritDoc}
```

Comment:
We do not need to use `inheritDoc` since we do not override the java doc comment here

Reply:
Fixed in bfc171f